temperature/models.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import Column, Integer, DateTime, Float, ForeignKey
+from sqlalchemy.orm import relationship
+
+from city.models import City
+from database import Base
+
+
+class Temperature(Base):
+    __tablename__ = "temperature"
+
+    id: int = Column(Integer, primary_key=True, index=True)
+    date_time = Column(DateTime, nullable=False)
+    temperature = Column(Float, nullable=False)
+    city_id = Column(Integer, ForeignKey("city.id"))
```

Comment:
Ensure that the 'city_id' field in the 'Temperature' model is correctly set up as a foreign key referencing the 'id' field in the 'City' model. This is crucial for maintaining the relationship between temperature records and cities.